megamek/src/megamek/MegaMek.java

```diff
@@ -94,7 +94,7 @@ public static void main(String[] args) {
                 }
             }
 
-            configureLogging(logFileName);
+//            configureLogging(logFileName);
```

Comment:
Is this commented out code something that needs to be retained?

Reply:
Ouch! Good you saw that. I always do that to get errors in the console. Corrected it.